edx_proctoring/templates/practice_exam/error.html

```diff
@@ -25,17 +25,20 @@ <h4>
     {% endblock %}
   </p>
   <hr>
-  <button class="gated-sequence start-timed-exam action-primary" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
-    <span class="icon fa fa-lock" aria-hidden="true"></span>
-    <a>
-      {% block retry_cta %}
+  {% block retry_cta %}
+    <button class="gated-sequence start-timed-exam action-primary" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
```

Comment:
Is there a reason we can't go through the same "reset_attempt" code path for these non-onboarding exams? I'm thinking the bug could still exist when called from this view unless we make backend changes to delete when this button is clicked.

Reply:
I think technically they can, and it may be better long term so that practice + onboarding exams have an even more similar flow